src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import { getUserById } from '../../services/user';
+import { completeTodo } from '../../services/todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user = getUserById(todo.userId);
+
+  return (
+    <article
+      className={`TodoInfo ${completeTodo(todo.completed)}`}
+      key={todo.id}
+      data-id={todo.id}
+    >
+      <h2 className="TodoInfo__title">{`${todo.title}`}</h2>
+      <UserInfo user={user} />
```

Comment:
```suggestion
      {user &&<UserInfo user={user} />}
```
so you can add the check here, and render the component only when user exists :)

Reply:
done